common/lib/xmodule/xmodule/tests/test_xblock_wrappers.py

```diff
@@ -49,7 +48,6 @@
 # TODO: Add more types of sample data
 LEAF_XMODULES = {
     AnnotatableDescriptor: [{}],
-    CapaDescriptor: [{}],
```

Comment:
This are 4 related tests:

1. `TestStudentView`: Not needed since `student_view` is implement by `ProblemBlock`.
2. `TestStudioView`: Not needed since `studio_view` is implement by `ProblemBlock`.
3. `TestPublicView`: `ProblemBlock` uses the default placeholder `public_view` so not that important a test.